Gems/MiniAudio/Code/Source/Clients/MiniAudioListenerComponentController.cpp

```diff
@@ -66,6 +66,28 @@ namespace MiniAudio
         return m_config;
     }
 
+    float MiniAudioListenerComponentController::GetGlobalVolumePercentage() const
+    {
+        return MiniAudioInterface::Get()->GetGlobalVolume()*100.f;
```

Comment:
For these mathematical operations, you should apply clang formatting rules found in the [clang-format ](https://github.com/o3de/o3de/blob/development/.clang-format) rules at the root of the project. You can refer to the [contribution guide](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md).

Reply:
I applied the clang-format rules to all MiniAudio source files, see: 73bca6ffc2e6fdecfa393f06ab29acf7f97f685d